mycore-restapi/src/main/java/org/mycore/restapi/v1/utils/MCRRestAPIObjectsHelper.java

```diff
@@ -580,9 +581,20 @@ public Date getLastModified() {
                 MCRDerivate der = MCRMetadataManager.retrieveMCRDerivate(MCRObjectID.getInstance(oid.getId()));
                 String mcrID = der.getDerivate().getMetaLink().getXLinkHref();
                 eDerObject.setAttribute("metadata", mcrID);
-                if (der.getLabel() != null) {
+                if (der.getLabel() != null && der.getLabel().length() > 0) {
                     eDerObject.setAttribute("label", der.getLabel());
                 }
+                if (der.getDerivate().getClassifications().size() > 0) {
+                    StringBuffer c = new StringBuffer();
+                    for (int i = 0; i < der.getDerivate().getClassifications().size(); i++) {
+                        if (i > 0) {
+                            c.append(" ");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid appending characters as strings in StringBuffer.append.](https://app.codacy.com/app/MyCoRe/mycore/pullRequest?prid=4625414)